BitPay/BitPay.cs

```diff
@@ -19,6 +19,7 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using Microsoft.Extensions.Configuration;
+using Org.BouncyCastle.Math;
```

Comment:
What is this one used for?

Reply:
To use the private key as a BigInterger.

Reply:
> What is this one used for?

To use the private key as a BitInteger.

Reply:
But I don't see where this is being used in this class